backend/prisma/schema.prisma

```diff
@@ -153,3 +154,13 @@ model PlanMembership {
   inviteStatus InviteStatus
   role         Role
 }
+
+model PlanInvite {
+  id        String   @id @default(uuid())
+  plan      Plan     @relation(fields: [planID], references: [id])
+  planID    String
+  inviteeEmail String
```

Comment:
I've been abset in a few convos. the Invitee Does't need to have a wewap account right? So we don't need to make a FK relation to the User's table here?